packages/react-components/src/components/SendBoxErrorBar.tsx

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { MessageBar, MessageBarType } from '@fluentui/react';
+import React, { useEffect } from 'react';
+
+/**
+ * @internal
+ */
+export interface SendBoxErrorBarProps {
+  /** Error message to render */
+  message?: string;
+  /**
+   * Timeout in ms after which the error bar disappears.
+   * If undefined, the error bar will never disappear.
+   */
+  timeout?: number;
+}
+
+/**
+ * @internal
+ */
+export const SendBoxErrorBar = (props: { message?: string; timeout?: number }): JSX.Element => {
+  const { message, timeout } = props;
+  const [errorMessage, setErrorMessage] = React.useState(message);
+
+  useEffect(() => {
+    if (timeout !== undefined) {
+      const messageTimeout = setTimeout(() => {
+        setErrorMessage(undefined);
+      }, timeout);
+      return () => {
+        clearTimeout(messageTimeout);
+      };
+    } else {
+      return;
```

Comment:
nit: remove empty else

Reply:
Added it to remove the warning, `not all code paths return`